iOS/Flows/Conversation/Manager/ConversationCollectionViewManager.swift

```diff
@@ -51,24 +51,10 @@ class ConversationCollectionViewManager: NSObject, UITextViewDelegate, Conversat
         self.collectionView = collectionView
         super.init()
         self.updateLayoutDataSource()
-
-//        ConversationSupplier.shared.$activeConversation.mainSink { [weak self] (conversation) in
-//            guard let `self` = self, let activeConversation = conversation else { return }
-//            switch activeConversation.conversationType {
-//            case .conversation(let conversation):
-//                conversation.getMembersCount { (result, count) in
-//                    self.numberOfMembers = Int(count)
-//                }
-//            default:
-//                break
-//            }
-//        }.store(in: &self.cancellables)
-
-//        self.collectionView.panGestureRecognizer.addTarget(self, action: #selector(handle(_:)))
```

Comment:
Removing code related to conversation details. It doesn't really make sense in this ThreadViewController.